webapp/package.json

```diff
@@ -67,7 +67,8 @@
   "devDependencies": {
     "@babel/core": "~7.12.3",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/preset-env": "~7.9.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.20.11",
```

Comment:
is this package really needed or an artifact from the suggestion I made/or a merge thing?

Reply:
I do not remember

Reply:
Does it work without the package?